tests/integration/targets/eos_facts/tests/cli/default_facts.yaml

```diff
@@ -12,7 +12,7 @@
       - "'default' in result.ansible_facts.ansible_net_gather_subset"
       - "'config' not in result.ansible_facts.ansible_net_gather_subset"
       - result.ansible_facts.ansible_net_hostname is defined
-      - result.ansible_facts.ansible_net_image is defined
+      # - result.ansible_facts.ansible_net_image is defined
```

Comment:
Can you please add a comment here stating why we're going this?

Reply:
Done!